app/models/returning_teacher.rb

```diff
@@ -1,8 +1,23 @@
 class ReturningTeacher < Base
   attribute :returning_to_teaching, :boolean
+  attribute :degree_options, :string
+  attribute :preferred_education_phase_id, :integer
+
+  DEGREE_OPTIONS = { returner: "returner" }.freeze
+
+  before_validation :set_degree_options
```

Comment:
Instead of doing these in a `before_validation` I think you could just default them as they are static:

```
attribute :degree_options, :string, default: -> { DEGREE_OPTIONS[:returner] }
attribute : preferred_education_phase_id, :integer, default: -> { StageInterestedTeaching::OPTIONS[:secondary].to_i }
```

Reply:
Can do the same with `attribute :preferred_education_phase_id, :integer, default: StageInterestedTeaching::OPTIONS[:secondary].to_i`

Reply:
default takes a block or a string, heh ho...